src/luxonis_ml/data/utils/data_utils.py

```diff
@@ -275,11 +245,24 @@ def generate_hashname(filepath):
     # Generate the UUID5 based on the file contents and the NAMESPACE_URL
     file_hash_uuid = uuid.uuid5(uuid.NAMESPACE_URL, file_contents.hex())
 
-    return str(file_hash_uuid) + os.path.splitext(filepath)[1]
+    return str(file_hash_uuid) + os.path.splitext(filepath)[1], str(file_hash_uuid)
 
 
 def is_modified_filepath(dataset, filepath):
     if filepath.startswith(f"/{dataset.team_id}"):
         return True
     else:
         return False
+
+
+def get_group_from_sample(dataset, sample):
+    group = sample[dataset.source.name]
+    group = dataset.fo_dataset.get_group(group["id"])
+    return group
+
+
+def get_filepath_from_hash(dataset, hash):
```

Comment:
What if instance wouldn't exist? Would it make sense to output some generic path in that case?

Reply:
I just edited it to return `None` now in that case

Reply:
Should we throw some error if it happens?